com.microsoft.java.debug.plugin/src/main/java/com/microsoft/java/debug/plugin/internal/eval/JdtEvaluationProvider.java

```diff
@@ -112,7 +112,6 @@ public void initialize(IDebugAdapterContext context, Map<String, Object> props)
             JDIThread jdiThread = getMockJDIThread(thread);
             JDIStackFrame stackframe = createStackFrame(jdiThread, depth);
             if (stackframe == null) {
-                logger.severe("Cannot evaluate because the stackframe is not available.");
```

Comment:
use throw AdapterUtils.createCompletionException?

Reply:
The code line `116` will throw an IllegalStateException instead of CompletionException, this is OK for the provider class, and the caller (e.g. Breakpoint Handler) has logic to explicitly catch the exception and log the error info.